swupd/create_manifests.go

```diff
@@ -110,14 +110,14 @@ func initBundles(ui UpdateInfo, c config) ([]*Manifest, error) {
 			}
 			if err != nil {
 				errorChan <- err
-				break
+				return
 			}
 
 			// detect type changes
 			// fail out here if a type change is detected since this is not yet supported in client
 			if bundle.hasUnsupportedTypeChanges() {
 				errorChan <- errors.New("type changes not yet supported")
-				break
+				return
```

Comment:
Kind-of +1 to these. It doesn't matter if we just break out of processing this or not IMO, cause if we write an error it should exit in the main routine anyway. BUT, this is fine here, maybe add a comment explicitly saying that the goroutine itself is no longer usable, which is not the case we want in other code paths which should continue on error.

Reply:
You don't break out of processing, you just move to the next bundle. But once an error occurs, mixer should stop.